kamrons-game.py

```diff
@@ -9,6 +9,14 @@ def typewriter(str):
         sleep(0.07)
     print("")
 
+
+def typewriter2(str):
+    for letter in str:
+        sys.stdout.write(letter)
+        sys.stdout.flush()
+        sleep(0.9)
+    print("")
+
 def get_answer(str):
     
     typewriter(str)
```

Comment:
I recommend using an outside `typewriter` function that takes two parameters. The first being the string to type and the second being the delay in between:
`def typewriter(str, delay):`

D.R.Y.!

Reply:
done